### ISSUE TITLE
When responding to a toot, allow to put context (quote/reply)

### ISSUE THREAD

**Issue by fwolfst** [tootsuite/mastodon#13876]
### Pitch

When answering to a toot, I miss the feature to "quote" parts of the original toot that I am referring to. Very similar to E-Mails or the way it can be done in discourse (mark text, klick "quote", done).

I have to admit that I am maybe just too blind to find this feature, if so - it is not obvious enough how to do that.

### Motivation

Meaningful discussion and satisfactory exchange is apparently supported if two engaging parties relate to each others claims.

**Comment by ThibG** [comment]
There is indeed no such feature. Common usage is simply to use textual markers such as actual quotes, or prefixing the lines with something like “>”.

There has also been discussion about allowing some form of formatting (bold text, italics, quote blocks, etc.) but so far it has been rejected.

**Comment by fwolfst** [comment]
I am happy if you put some links to that discussion. I believe it would improve overall discussion and statement-quality.

**Comment by shleeable** [comment]
This is one of those "debates"... the dev team don't like it

**Comment by ThibG** [comment]
@fwolfst https://github.com/tootsuite/mastodon/pull/10629

**Comment by fwolfst** [comment]
> @fwolfst #10629

Gosh, I see. Thanks a lot. Unfortunately I don't have time to consume the whole discussion, but skimming through it I see why and what is happening :)
I'd love to chime in there and would have some qualifications to do so, but lack the resources.

**Comment by realpixelcode** [comment]
Yeah, something like that would be very nice because manually quoting makes everything a bit cluttered.

**Comment by brendanjones** [comment]
Yes to this. Helps to respond to individual points within a longer post.

Basically, markdown is needed: https://github.com/mastodon/mastodon/issues/18958

**Comment by diondiondion** [comment]
Consent-respecting quote posts were introduced in [Mastodon 4.5](https://github.com/mastodon/mastodon/releases/tag/v4.5.0) so we're closing this issue now. If you have any concerns or feedback about the new quote posts, please open a new issue.

You can learn more about Quote posts in our [blog post](https://blog.joinmastodon.org/2025/09/introducing-quote-posts/), our [FAQ](https://docs.joinmastodon.org/user/quote-posts/#faq), and our [developer implementation guide](https://docs.joinmastodon.org/client/quotes/).

**Comment by trwnh** [comment]
> feature to "quote" parts of the original toot

this feature seems to be asking for something different than the quote posts feature introduced in mastodon 4.5, which does not allow quoting parts or excerpts.
